collector/runtime-benchmarks/hashmap/src/main.rs

```diff
@@ -1,11 +1,11 @@
 use benchlib;
-use benchlib::benchmark::benchmark_suite;
+use benchlib::benchmark::benchmark_group;
 use benchlib::define_benchmark;
 
 fn main() {
-    benchmark_suite(|suite| {
+    benchmark_group(|group| {
         // Measures how long does it take to insert 10 thousand numbers into a `hashbrown` hashmap.
-        define_benchmark!(suite, hashmap_insert_10k, {
+        define_benchmark!(group, hashmap_insert_10k, {
```

Comment:
`define_benchmark` is a name that leaves me wondering exactly what the macro does. I think it effectively defines *and* runs the benchmark? Should it become `run_benchmark!`?  Or should `benchmark_group` becomes `define_benchmark_group` (as opposed to `run_benchmark_group` which I suggested above). Either way, consistency would be good.

Reply:
Also, I noticed that `BenchmarkGroup::register` has a comment mentioning `func`, but I think it should instead mention `constructor`.

Reply:
It indeed only defines the benchmark. The benchmark is not executed by itself immediately. The `run_benchmark_group` function scans CLI arguments and decides whether to run benchmarks, list them to output etc.